packages/protocol/test-sol/governance/network/GovernanceSlasher.t.sol

```diff
@@ -0,0 +1,120 @@
+// SPDX-License-Identifier: UNLICENSED
+pragma solidity ^0.5.13;
+
+import "celo-foundry/Test.sol";
+
+import "../../../contracts/common/Accounts.sol";
+import "../../../contracts/common/FixidityLib.sol";
+import "../../../contracts/common/interfaces/IRegistry.sol";
+import "../../../contracts/governance/Proposals.sol";
+import "../../../contracts/governance/test/MockLockedGold.sol";
+import "../../../contracts/governance/test/MockValidators.sol";
+
+// Contract to test
+import "../../../contracts/governance/GovernanceSlasher.sol";
+
+contract GovernanceSlasherTest is Test {
+  event SlashingApproved(address indexed account, uint256 amount);
+  event GovernanceSlashPerformed(address indexed account, uint256 amount);
+
+  IRegistry registry;
+  Accounts accounts;
+  MockLockedGold mockLockedGold;
+  MockValidators mockValidators;
+
+  GovernanceSlasher public governanceSlasher;
+  address owner;
+  address nonOwner;
+  address validator;
+  address slashedAddress;
+
+  address registryAddress = 0x000000000000000000000000000000000000ce10;
+
+  function setUp() public {
+    owner = address(this);
+    nonOwner = actor("nonOwner");
+    validator = actor("validator");
+    slashedAddress = actor("slashedAddress");
+
+    accounts = new Accounts(true);
+    mockLockedGold = new MockLockedGold();
+    mockValidators = new MockValidators();
```

Comment:
Are you sure you need the mocked version? I couldn't find a reference

Reply:
I was keeping it similar to the original test. Changed it to use the actual contract.